Functions.Templates/Templates/HttpTrigger-FSharp/HttpTriggerFSharp.fs

```diff
@@ -43,8 +43,10 @@ module HttpTriggerFSharp =
                    | null -> ""
                    | nc -> nc.Name
             
+            let mutable responseMessage = "This HTTP triggerred function executed successfully. Pass a name in the query string or in the request body for a personalized response."
+            
             if not (String.IsNullOrWhiteSpace(name)) then
-                return OkObjectResult(sprintf "Hello, %s" name) :> IActionResult
-            else
-                return BadRequestObjectResult("Please pass a name on the query string or in the request body") :> IActionResult
-        } |> Async.StartAsTask
+                responseMessage <- "Hello" +  name + ". This HTTP triggerred function executed successfully"
```

Comment:
This can be rewritten in a more idiomatic way:

```fsharp
let responseMessage =
    if not (String.IsNullOrWhiteSpace(name)) then
        "Hello, " + "name" + ". This HTTP triggerred function executed successfully."
    else
        "This HTTP triggerred function executed successfully. Pass a name in the query string or in the request body for a personalized response."
```

Reply:
Fixed